Sources/JWTKit/JWK/JWK.swift

```diff
@@ -4,67 +4,95 @@ import class Foundation.JSONDecoder
 /// A JSON Web Key.
 ///
 /// Read specification (RFC 7517) https://tools.ietf.org/html/rfc7517.
-public struct JWK: Decodable {
+public struct JWK: Codable {
     /// Supported `kty` key types.
-    public enum KeyType: Decodable {
+    public enum KeyType: String, Codable {
         /// RSA
         case rsa
-        
+        /// EC
+        case ec
```

Comment:
Is this the technically correct name? I thought it was "ecdsa". We should double check this